locales/en/messages.json

```diff
@@ -1332,7 +1332,7 @@
         "message": "IRC Tramp"
     },
     "portsFunction_RUNCAM_DEVICE_CONTROL": {
-        "message": "RunCam Device"
+        "message": "RunCam/Caddx Device"
```

Comment:
Not sure about this - if more manufacturers start supporting this protocol then we'll have to change this again. I'd suggest we go for something like 'Camera device (RunCam protocol)'.

Thoughts? @azolyoung?

Also, note that our translations are done with an external tool (crowdin) by a group of volunteers. So it is not necessary to supply any translation except for English, the others will be translated in the tool, and the language files overwritten by the tool's output before every release.